azdev/operations/code_gen.py

```diff
@@ -55,24 +60,25 @@ def create_module(mod_name='test', display_name=None, display_name_plural=None,
     _display_success_message(COMMAND_MODULE_PREFIX + mod_name, mod_name)
 
 
-def create_extension(ext_name='test', repo_name='azure-cli-extensions',
-                     display_name=None, display_name_plural=None,
-                     required_sdk=None, client_name=None, operation_name=None, sdk_property=None,
-                     not_preview=False, github_alias=None, local_sdk=None):
-    repo_path = None
+def create_extension(ext_name, azure_rest_api_specs=const.GITHUB_SWAGGER_REPO_URL, branch=None, use=None):
+    if not azure_rest_api_specs.startswith('http') and branch:
+        raise CLIError('Cannot specify azure-rest-api-specs repo branch when using local one.')
+    if not branch:
+        branch = 'master'
```

Comment:
May not worth changing it, but you can get `default_branch` with https://api.github.com/repos/Azure/azure-rest-api-specs

Reply:
Good suggestion! I will update it in my next commit